### PR TITLE
fix #65697 topnewsshow.com

### LINKED ISSUE #65697: topnewsshow.com
### Issue URL (Anti Adblock Script)
 [https://topnewsshow.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Ftopnewsshow.com%2F)
### Comment
Username: @nickv
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.12
Browser: | Chrome
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances, <br/>Adblock Warning Removal List<br/><br/><b>Other:</b><br/> Filter unblocking search ads and self-promotion<br/><br/><b>Language-specific:</b><br/> AdGuard French, <br/>AdGuard Russian, <br/>AdGuard Japanese, <br/>AdGuard Spanish/Portuguese, <br/>ABPindo, <br/>EasyList Czech and Slovak, <br/>EasyList Italy, <br/>Liste AR, <br/>List-KR

### PR DESCRIPTION
**Pull request by stanislav-atr**
#65697

### CODE REVIEW

File: EnglishFilter/sections/specific.txt [hunk 1]
```diff
@@ -425,6 +425,7 @@ biguz.net##.topb
 biguz.net###under
 subtitletools.com##.hide-when-adblock
 subtitletools.com##a[href^="https://go.nordvpn.net/"][target="_blank"]
+topnewsshow.com###content  > div#secondary > aside#custom_html-4
```
Comment (zloyden) [inline EnglishFilter/sections/specific.txt:428]: `topnewsshow.com###secondary > aside#custom_html-4` this is enough.

File: FrenchFilter/sections/antiadblock.txt [hunk 2]
```diff
@@ -26,6 +26,9 @@ hollywoodpq.com#%#//scriptlet("set-constant", "ujloijdkhjkwus", "false")
 monacomatin.mc#@##adsense
 monacomatin.mc#@#.AdSense
 monacomatin.mc#%#//scriptlet("abort-current-inline-script", "$", "checkAds")
+! https://github.com/AdguardTeam/AdguardFilters/issues/65697
+topnewsshow.com#@#.an-sponsored
+topnewsshow.com#@#.an-advert-banner
```
Comment (zloyden) [inline FrenchFilter/sections/antiadblock.txt:31]: I don't see the adblock detection. Do you still have it?

Comment (Sergey-Lyapin) [reply FrenchFilter/sections/antiadblock.txt:31]: I've reproduce it, It is caused by anOptions script, please add to the appropriate section.

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: FrenchFilter/sections/antiadblock.txt (hunk 2 above)
Comment (zloyden) [reply FrenchFilter/sections/antiadblock.txt:31]: Hm, in this case remove those rules and add the domain to the `anOptions` section in the Base filter. If you can close the message, then only add it to the Annoyances.

**Comment by zloyden** [comment]
According to the situation, I made changes directly.

**Pull request closed without merging**